docs/components/modeler/web-modeler/collaboration.md

```diff
@@ -28,6 +28,10 @@ There are four roles with different levels of access rights that can be assigned
 
 ### Inviting users to projects
 
+:::note
+Web Modeler expects users to have email addresses to receive invitations correctly.
```

Comment:
You originally said "expects to have email addresses **assigned**" here -- do you mean they just need an email address, or an email address assigned to them within Modeler?

Reply:
@christinaausley, thanks for your question!
They need an email address associated with their account in the identity management system (for instance, Keyclock in the self-managed context). They can join the current project with the invitation, so the Modeler itself. To receive an invitation, you'll need an email address.
Please let me know if what I wrote makes sense and if you need further clarification. Feel free to propose a more descriptive note here in the doc.

Reply:
@christinaausley done with https://github.com/camunda/camunda-platform-docs/pull/1793/commits/ed1e69359fccd3b276d293a2737ba0ca142bf0a1